gateway/src/apicast/policy/upstream_connection/apicast-policy.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+  "name": "Upstream connection",
+  "summary": "Allows to configure several options for the connections to the upstream",
+  "description": "Allows to configure several options for the connections to the upstream",
+  "version": "builtin",
+  "configuration": {
+    "type": "object",
+    "properties": {
+      "connect_timeout": {
+        "description": "Timeout for establishing a connection (in seconds).",
+        "type": "integer"
+      },
+      "send_timeout": {
```

Comment:
Just a note here, could we use a type number with a range? Like this for example:

https://json-schema.org/understanding-json-schema/reference/numeric.html#range

So validation for negative numbers can be in the schema.